great_tables/_styles.py

```diff
@@ -220,12 +220,23 @@ class CellStyleText(CellStyle):
     ) = None
 
     def _to_html_style(self) -> str:
+
         rendered = ""
 
         if self.color:
             rendered += f"color: {self.color};"
         if self.font:
-            rendered += f"font-family: {self.font};"
+            font = self.font
```

Comment:
Can you test the lines here that are flagged as untested in codecov? 

app.codecov.io/gh/posit-dev/great-tables/pull/423?dropdown=coverage&src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=checks&utm_campaign=pr+comments&utm_term=posit-dev